docs/commerce/order_management/configure_order_management.md

```diff
@@ -41,4 +41,10 @@ Then reference it with `ibexa.repositories.<your_repository>.order_management.wo
 
 ``` yaml
 [[= include_file('code_samples/front/shop/order-management/config/packages/ibexa.yaml', 69, 74) =]]
-```
\ No newline at end of file
+```
+
+#### PIM integration
+
+By default, the component integration mechanism reduces product stock values when an order is confirmed and reverts it to the original value when an order is cancelled.
```

Comment:
Feel free to rephrase it, the point is that stock is reduced once the order is made, not confirmed (it was changed within https://github.com/ibexa/order-management/pull/45).
```suggestion
By default, the component integration mechanism reduces product stock values when an order is made (in status "pending") and reverts it to the original value when an order is cancelled.
```